StarterKit/Session1/agenda.md

```diff
@@ -78,7 +81,7 @@ Instantiation
 
 ## Content
 
-The goal of this exercise is to start to think about which lines in the code define it's essence, have the biggest impact or need to be paid close attention to.
+The goal of this exercise is to start to think about which lines in the code define its essence, have the biggest impact or need to be paid close attention to.
 
 ### (10 mins) Independently - Identify most important lines
```

Comment:
I think 10 minutes might be too much for this

Reply:
Change to 8? Sometimes the code is harder to parse than others, but stick at 8.

Maybe we should make a general guidance about the facilitator using the timings as approx to aim for.